### PR TITLE
Add stylelint

### PR DESCRIPTION
**Pull request by hzxcaryn**
[Stylelint](https://stylelint.io/) lints CSS code. Since we have agreed on using [styled components](https://styled-components.com/), I have configured stylelint for that usecase.
Also added scripts for CSS linting and edited the README accordingly.

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -28,7 +28,9 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "gts check",
+    "lint": "gts check && yarn lint:css",
```
Comment (karenfrilya97) [inline web/package.json:31]: I think this one should standardise, either `yarn lint:code && yarn lint:css` or `gts check && stylelint ./src/**/*.tsx`. I think the first one is better cos you don't have to change twice if you change anything.

File: web/README.md [hunk 2]
```diff
@@ -20,7 +20,15 @@ You will also see any lint errors in the console.
 
 ### `yarn lint`
 
-Lints and check for formatting problems according to [Google Typescript Style](https://github.com/google/gts).
+Lints and checks code for formatting problems according to [Google Typescript Style](https://github.com/google/gts). Lints CSS as well.
```
Comment (karenfrilya97) [inline web/README.md:23]: should we add a link for the CSS linter also?

File: web/package.json (hunk 1 above)
Comment (hzxcaryn) [reply web/package.json:31]: Oh yea! Didn't realise this haha thanks!

File: web/README.md (hunk 2 above)
Comment (hzxcaryn) [reply web/README.md:23]: @karenfrilya97 Hmmmm I'm not sure, gts comes with its set of rules (aka opinionated in some ways), but for the css linter i'm just using the standard lint rules so I'm not so sure hmm.
@ifhuang @shengdac @erencie  Any inputs on this? 😛

File: web/README.md [hunk 3]
```diff
@@ -20,7 +20,15 @@ You will also see any lint errors in the console.
...
+
+#### `yarn lint:code`
+
+Lints code according to [Google Typescript Style](https://github.com/google/gts) only.
+
+#### `yarn lint:css`
+
+Lints CSS. Run ``yarn lint:css --fix`  to fix lint errors automatically (if possible).
```
Comment (ifhuang) [inline web/README.md:31]: extra `?

File: web/README.md (hunk 2 above)
Comment (ifhuang) [reply web/README.md:23]: Actually why do we need this stylelint? gts doesn't cover it?

Comment (hzxcaryn) [reply web/README.md:23]: @ifhuang Nope, gts is not catered towards frontend dev specifically so CSS linter is not included! gts uses eslint and prettier behind the hood, those tools lint and format code, but not styles!

File: web/README.md (hunk 3 above)
Comment (hzxcaryn) [reply web/README.md:31]: Good catch! My .md files are bugging so bad on my vscode I didn't even realise this 😭 😢

Comment (hzxcaryn) [reply web/README.md]: Fixed it!

File: web/README.md (hunk 2 above)
Comment (ifhuang) [reply web/README.md:23]: ic. adding a link for the CSS linter SGTM

Comment (hzxcaryn) [inline web/README.md:23]:
```suggestion
Lints and checks TypeScript code for formatting problems according to [Google Typescript Style](https://github.com/google/gts), and lints CSS code inside styled components using [stylelint](https://stylelint.io/) with the [standard config rules](https://github.com/stylelint/stylelint-config-standard).
```

File: web/README.md [hunk 4]
```diff
@@ -20,7 +20,15 @@ You will also see any lint errors in the console.
 
 ### `yarn lint`
 
-Lints and check for formatting problems according to [Google Typescript Style](https://github.com/google/gts).
+Lints and checks code for formatting problems according to [Google Typescript Style](https://github.com/google/gts). Lints CSS as well.
+
+#### `yarn lint:code`
+
+Lints code according to [Google Typescript Style](https://github.com/google/gts) only.
```
Comment (hzxcaryn) [inline web/README.md:27]:
```suggestion
Lints and checks TypeScript code for formatting problems according to [Google Typescript Style](https://github.com/google/gts) only.
```

File: web/README.md [hunk 5]
```diff
@@ -20,7 +20,15 @@ You will also see any lint errors in the console.
...
+
+#### `yarn lint:css`
+
+Lints CSS in styled components.
```
Comment (hzxcaryn) [inline web/README.md:31]:
```suggestion
Lints CSS in styled components using [stylelint](https://stylelint.io/) with the [standard config rules](https://github.com/stylelint/stylelint-config-standard).
```

**Pull request merged**